packages/analytics-nextjs/src/events/story.ts

```diff
@@ -1,35 +0,0 @@
-export const STORY_ASSET = {
-    CLICK: 'Story Asset Click',
-};
-
-export const STORY_CONTACT = {
-    CLICK: 'Story Contact Click',
-};
-
-export const STORY_EMBED = {
-    CLICK: 'Story Embed Click',
-};
-
-export const STORY_FILE = {
-    CLICK: 'Story File Click',
-    DOWNLOAD: 'Story File Download',
-};
-
-export const STORY_GALLERY_IMAGE = {
-    DOWNLOAD: 'Gallery Image Download',
-    VIEW: 'Gallery Image View',
-};
-
-export const STORY_IMAGE = {
-    CLICK: 'Story Image Click',
-    DOWNLOAD: 'Story Image Download',
-    VIEW: 'Story Image View',
```

Comment:
@alpower Previously we've also tracked when image/gallery image was "opened" in stories. Not sure if we need this in Plausible? 🤔

Reply:
would it hurt to leave it in? I can see this being nice to know, if not now, at some point.

Reply:
I mean.. I guess we could keep it, just seems a bit useless to track that if we're not gonna show it in the dashboard.

Reply:
If it's not costing us anything to track it I say keep it, as quite what we show in the dashboard is still up in the air a bit.

Reply:
Custom events count as pageviews - we have 10 M events per month and we're already at 0,5 M and it's the 4th day of the month, so in theory it should be okay, but who knows, depends on the amount of images visitors are viewing.

Also, even if we add the Goals to the dashboard later, it won't show past data so I'd say let's skip it for now and I can add it later if we decide that we want this information there, otherwise let's not waste the event limit.

Sounds good?

Reply:
yeah sounds good - if it counts towards limits then I agree.